clients/venice-client/src/main/java/com/linkedin/venice/fastclient/ClientConfig.java

```diff
@@ -96,7 +106,8 @@ private ClientConfig(
       boolean isVsonStore,
       StoreMetadataFetchMode storeMetadataFetchMode,
       D2Client d2Client,
-      String clusterDiscoveryD2Service) {
+      String clusterDiscoveryD2Service,
+      boolean batchGetDefaultsToStreamingBatchGet) {
```

Comment:
NIT: the variable sounds a little verbose, maybe rename it to `useStreamingBatchGetAsDefault`?

Reply:
good suggestion, renamed it.